blocks/procedures.ts

```diff
@@ -76,10 +94,10 @@ const PROCEDURE_DEF_COMMON = {
   },
   /**
    * Update the display of parameters for this procedure definition block.
-   * @private
-   * @this {Block}
+   *
+   * @internal
```

Comment:
If the intention is to migrate `@private` methods to `@internal` methods please do this throughout the rest of the file!

Reply:
Done.  I've also gone back to check the others (there are a few missing in `text.ts`), which I will file a separate PR about.